src/pages/resultsView/cancerSummary/CancerSummaryContainer.tsx

```diff
@@ -79,21 +79,27 @@ export default class CancerSummaryContainer extends React.Component<
         }
     }
 
-    // this is used to map study id to study shortname
-    private mapStudyIdToShortName(str: string) {
+    // this is used to map study id to study name
+    private mapStudyIdToName(str: string) {
         if (str in this.props.store.physicalStudySet) {
-            return this.props.store.physicalStudySet[str].shortName;
+            const studyname = this.props.store.physicalStudySet[str].name;
+            if (studyname.indexOf('(') > 50) {
```

Comment:
you should make this a helper function and write a unit test for it

Reply:
I created a helper function. Let me know which file to place it.

Reply:
fixed and added into FormatUtils. added unit tests.